src/plugins/so_management/server/index.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
```

Comment:
the plugin name is `savedObjectsManagement`. Should I rename the plugin folder from `so_management` to `saved_objects_management` ?

Reply:
Yea, that's probably the right call, as the folder should probably match the plugin name. Gonna rename